### DISCUSSION TITLE
share with obtanium

### DISCUSSION THREAD

**Discussion by zen0bit** [ImranR98/Obtainium#879] [Ideas]
i would like to have option to share copied links with obtanium directly

now i must:
 copy link (options to choose what you can do with it opens) 

choose copy link because obtanium isn't in options.

switch to obtanium

choose add new app

paste link

suggestions:
copy link
share with obtanium

ps: love Obtanium ❤️
keep up good work 👍

**Comment by zen0bit** [comment]
not sure if is same on all UIs
samsung OneUi

**Comment by ImranR98** [comment]
This should just be an issue, not a discussion. And it already is an issue (#109).
